tools/migrations/remove_workshop_demos.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+
+import datetime
+import json
+import os
+import sys
+import requests
+import argparse
+import sqlite3 as lite
+
+
+# parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("--demoinfo_db", "-demoinfodb", required=True, help="databse directory containing 'demoinfo.db' file")
+ap.add_argument("--blob_db", "-blobsdb", required=True, help="databse directory containing 'blobs.db' file")
+ap.add_argument("--blob_dir", "-blobsdir", required=True, help="staticData directory containing blobs")
+ap.add_argument("-r", '--remove', help="Remove Demo", action="store_true")
+args = ap.parse_args()
+demoinfo_db = args.demoinfo_db
+blob_db = args.blob_db
+blob_dir = args.blob_dir
+
+def connect_to_db(db_file):
+    try:
+        return lite.connect(db_file)
+    except Exception as ex:
+        print("Couldn't connect to the DB {}. Error: {}".format(db_file, ex))
+        raise
+
+def main():
+    di_conn = None
+    DI_DB = os.path.join(demoinfo_db, 'demoinfo.db')
+    BL_DB = os.path.join(blob_db, 'blobs.db')
+    try:
+        di_conn = connect_to_db(DI_DB)
+        di_cursor = di_conn.cursor()
+
+        bl_conn = connect_to_db(BL_DB)
+        bl_cursor = bl_conn.cursor()
+
+        demoinfo_demo_list = get_workshop_demos_from_demoinfo(di_cursor)
+
+        workshop_demo_list, old_workshop_demo_list, non_working_demos = ([] for i in range(3))
+        for workshop_demos in demoinfo_demo_list:
+            demo_id = workshop_demos['id']
+            workshop_demo_list.append(demo_id)
+
+            parameter = get_params(di_cursor, demo_id)
+
+            #input blob required to run the demo
+            blob = get_blob(bl_cursor, demo_id)
```

Comment:
My understanding is that some demos might have more than one blob required for running a demo. Running demos this way might lead to demo no executed due to missing inputs.